### PR TITLE
Changelog and other admin for 1.3 release

### PR DESCRIPTION
**Pull request by dbwiddis**

### CODE REVIEW

**Comment by hazendaz** [comment]
Sorry this will need cleaned up a bit.  The maven release takes care of versioning stuff so it has now been changed.

File: pom.xml [hunk 1]
```diff
@@ -10,7 +10,7 @@
     </parent>
     <groupId>com.github.dblock</groupId>
     <artifactId>oshi-core</artifactId>
-    <version>1.3-SNAPSHOT</version>
+    <version>1.3</version>
```
Comment (hazendaz) [inline pom.xml]: This is done via the maven release plugin so needs removed from this pull request.

**Comment by dbwiddis** [comment]
reverted the pom change. let me know anything else to change.

**Comment by hazendaz** [comment]
Looks good to me.  Merging...

**Pull request merged**